docs/tutorials/t-08.md

```diff
@@ -0,0 +1,127 @@
+## Objective
+
+This tutorial is to demonstrate [`Algosigner`](https://github.com/PureStake/algosigner) and [`Wallet Connect`](https://walletconnect.com/) in webapp using [`@algo-builder/web`](https://github.com/scale-it/algo-builder/tree/master/packages/web). [`@algo-builder/web`](https://github.com/scale-it/algo-builder/tree/master/packages/web) is designed to be used with web dapps as well as scripts and user programs and also allows you to interact with contracts easily.
+
+## What is `@algo-builder/web`?
+
+This package provides a class `WebMode` which has variety of high level functions like, [`waitForConfirmation`](algobuilder.dev/api/web/classes/web.html#waitForConfirmation), [`executeTx`](<(algobuilder.dev/api/web/classes/web.html#executeTx)>), [`signTransaction`](<(algobuilder.dev/api/web/classes/web.html#signTransaction)>), etc. These functions help sending transactions and building dapps.
```

Comment:
I can't asset to link `waitForConfirmation`, `executeTx` and `signTransaction`. Could you check it

Reply:
Have updated the link